### PR TITLE
[CodeClean] Apply regular expressions

### PR DESCRIPTION
**Pull request by abcinje**
Use regular expressions instead of combined if-statements.

related issue: nnsuite#1745

Signed-off-by: Injae Kang <abcinje@gmail.com>

### CODE REVIEW

File: gst/nnstreamer/tensor_common.c [hunk 1]
```diff
@@ -979,45 +979,50 @@ gst_tensor_get_type (const gchar * typestr)
...
-     * (case insensitive)
-     */
-    if (len == 6) {             /* uint16, uint32 */
-      if (type_string[4] == '1' && type_string[5] == '6')
+  if (g_regex_match_simple ("^uint(8|16|32|64)$",
+          type_string, G_REGEX_CASELESS, 0)) {
+    size = g_ascii_strtoull (&type_string[4], NULL, 10);
+
+    switch (size) {
+      case 8:
```
Comment (wooksong) [inline gst/nnstreamer/tensor_common.c]: How about use gst_tensor_get_element_size() instead of a hard-coded value?

Comment (myungjoo) [reply gst/nnstreamer/tensor_common.c]: Nope. ```gst_tensor_get_element_size()``` is a reverse function of what's needed here.
```gst_tensor_get_element_size()```: tensor_type --> size
what we need here: size --> tensor_type

Comment (wooksong) [reply gst/nnstreamer/tensor_common.c]: @myungjoo Ah! Got it. My bad.

File: gst/nnstreamer/tensor_common.c [hunk 2]
```diff
@@ -979,45 +979,50 @@ gst_tensor_get_type (const gchar * typestr)
...
-  if (type_string[0] == 'u' || type_string[0] == 'U') {
-    /**
-     * Let's believe the developer and the following three letters are "int"
-     * (case insensitive)
-     */
-    if (len == 6) {             /* uint16, uint32 */
-      if (type_string[4] == '1' && type_string[5] == '6')
+  if (g_regex_match_simple ("^uint(8|16|32|64)$",
+          type_string, G_REGEX_CASELESS, 0)) {
+    size = g_ascii_strtoull (&type_string[4], NULL, 10);
```
Comment (jaeyun-jung) [inline gst/nnstreamer/tensor_common.c]: function 'g_ascii_strtoull' returns the value type 'guint64'

Comment (myungjoo) [reply gst/nnstreamer/tensor_common.c]: need cast for 32b/64b safety? (in stricter systems, compilers may incur warnings/errors?)

**Pull request merged**